deps/rabbitmq_mqtt/test/v5_SUITE.erl

```diff
@@ -389,6 +401,21 @@ session_expiry_quorum_queue_disconnect_decrease(Config) ->
     ok = session_expiry_disconnect_decrease(rabbit_quorum_queue, Config),
     ok = rpc(Config, application, unset_env, [?APP, durable_queue_type]).
 
+zero_session_expiry_disconnect_autodeletes_qos0_queue(Config) ->
+    ClientId = ?FUNCTION_NAME,
+    C = connect(ClientId, Config, [
+        {clean_start, false},
+        {properties, #{'Session-Expiry-Interval' => 0}}]),
+    {ok, _, _} = emqtt:subscribe(C, <<"topic0">>, qos0),
+    QsQos0 = rpc(Config, rabbit_amqqueue, list_by_type, [rabbit_mqtt_qos0_queue]),
+    ?assertEqual(1, length(QsQos0)),
+
+    ok = emqtt:disconnect(C),
+    %% After terminating a clean session, we expect any session state to be cleaned up on the server.
+    timer:sleep(200), %% Give some time to clean up exclusive classic queue.
```

Comment:
Can we fix this comment? It's not an ` exclusive classic queue` that gets cleaned up, it's an MQTT QoS 0 queue.